### ISSUE TITLE
NullReferenceException reported in log in "OnProjectWindowItemGUI"

### ISSUE THREAD

**Issue by StanleyGoldman** [github-beta/unity-preview#31]
Version of Unity: All
Version of GitHub for Unity: 0.0.9.1
Platform: All

This error message was initially reported in #24 

```
NullReferenceException: Object reference not set to an instance of an object
GitHub.Unity.ProjectWindowInterface.OnProjectWindowItemGUI (System.String guid, Rect itemRect) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs:233)
UnityEditor.ObjectListArea+LocalGroup.DrawItem (Rect position, UnityEditor.FilterResult filterItem, UnityEditor.BuiltinResource builtinResource, Boolean isFolderBrowsing) (at C:/buildslave/unity/build/Editor/Mono/ObjectListLocalGroup.cs:855)
UnityEditor.ObjectListArea+LocalGroup.DrawInternal (Int32 beginIndex, Int32 endIndex, Single yOffset) (at C:/buildslave/unity/build/Editor/Mono/ObjectListLocalGroup.cs:192)
UnityEditor.ObjectListArea+Group.Draw (Single yOffset, Vector2 scrollPos) (at C:/buildslave/unity/build/Editor/Mono/ObjectListGroup.cs:144)
UnityEditor.ObjectListArea.HandleListArea () (at C:/buildslave/unity/build/Editor/Mono/ObjectListArea.cs:1367)
UnityEditor.ObjectListArea.OnGUI (Rect position, Int32 keyboardControlID) (at C:/buildslave/unity/build/Editor/Mono/ObjectListArea.cs:562)
UnityEditor.ProjectBrowser.OnGUI () (at C:/buildslave/unity/build/Editor/Mono/ProjectBrowser.cs:1754)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222)

```

**Comment by StanleyGoldman** [comment]
@polyren ~~Would you happen to know what file you right clicked?~~

Never mind, that seems to be based on file status.

**Comment by StanleyGoldman** [comment]
Resolved in v0.10-alpha
